scripts/secure-boot/generate-images/wrapper.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SCRIPTDIR=$(readlink -f $(dirname "$0"))
```

Comment:
```
In scripts/secure-boot/generate-images/wrapper.sh line 3:
SCRIPTDIR=$(readlink -f $(dirname "$0"))
                        ^-------------^ SC2046 (warning): Quote this to prevent word splitting.
```

Reply:
pre-commit now passes, changed [this line](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/bce10d72eb89079d32fb6db71846782fa2f3e593#diff-ed91284b570a2b939ac0f9b2f2920b30663fb3a865b8503338410983fd0a84c2R5)